main/runDiffer.sh

```diff
@@ -67,9 +86,19 @@ elif [[ -z "$remoteClusterName" ]];then
 	exit 1
 fi
 
+go build -o $execGo
+if (( $? != 0 ));then
+	echo "Unable to build xdcr diff tool"
+	exit 1
+fi
+
 export CBAUTH_REVRPC_URL="http://$username:$password@$hostname"
 echo "Exporting $CBAUTH_REVRPC_URL"
 
-$execGo -sourceUrl "$hostname" -sourceUsername $username -sourcePassword $password -sourceBucketName $sourceBucketName -targetBucketName $targetBucketName -remoteClusterName $remoteClusterName
+if [[ ! -z "$remoteClusterUsername" ]] && [[ ! -z "$remoteClusterPassword" ]];then
```

Comment:
should these be kept consistent with remoteClusterName? e.g., if remoteClusterName is empty, these should be empty too?